### PR TITLE
Igor lig 1835 add instructions to install docker

### PR DESCRIPTION
**Pull request by IgorSusmelj**
## Changes

- Refactor docs around downloading and installing the Lightly Worker
- Add docs to install docker with GPU support

### CODE REVIEW

File: docs/source/docker/getting_started/setup.rst [hunk 1]
```diff
@@ -42,83 +42,82 @@ See :ref:`rst-installing` for details.
...
+    We assume :code:`container-credentials.json` is in your current directory.
 
-    cat container-credentials.json | sudo docker login -u _json_key --password-stdin https://eu.gcr.io
+    .. code-block:: console
 
+        cat container-credentials.json | docker login -u _json_key --password-stdin https://eu.gcr.io
 
-**Third**, after authentication you should be able to pull our latest image. 
-Using the following command you pull the latest image from our European cloud server:
+    You should see a message stating `Loging Succeeded`.
```
Comment (guarin) [inline docs/source/docker/getting_started/setup.rst:90]: Loging -> Login

**Review by guarin — APPROVED** [review]
Awesome! Just one small typo.

File: docs/source/docker/getting_started/setup.rst [hunk 2]
```diff
@@ -42,83 +42,82 @@ See :ref:`rst-installing` for details.
...
 .. _docker-download-and-install:
 
-Download the Lightly Worker
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Docker Installation
+^^^^^^^^^^^^^^^^^^^^
 
-The Lightly worker comes as a Docker image. Ask your account manager from Lightly for the credentials
-to download the image.
+Lightly Worker requires docker to run. We highly recommend a docker Installation
```
Comment (japrescott) [inline docs/source/docker/getting_started/setup.rst:48]: Installation -> installation

File: docs/source/docker/getting_started/setup.rst [hunk 3]
```diff
@@ -42,83 +42,82 @@ See :ref:`rst-installing` for details.
...
+    docker could not select device driver with capabilities gpu
 
-.. code-block:: console
 
-    cat container-credentials.json | docker login -u json_key --password-stdin https://eu.gcr.io
+If you don't have docker installed or without GPU support we recommend following
+our guide about :ref:`rst-docker-known-issues-faq-install-docker`. 
 
+.. note::
+   If you use a cloud instance (e.g. on AWS, GCP or Azure) Docker is most likely
```
Comment (japrescott) [inline docs/source/docker/getting_started/setup.rst:68]: maybe make suggestion which "OS" should be selected when starting the machine (or do we have that info somewhere else?)

File: docs/source/docker/getting_started/setup.rst [hunk 4]
```diff
@@ -42,83 +42,82 @@ See :ref:`rst-installing` for details.
...
-#. give your user permission to run - recommended
-   docker (see https://docs.docker.com/engine/install/linux-postinstall/) 
-#. run Docker commands as root (always replace `docker` with `sudo docker`) - functional but less secure
+1. Make sure :code:`container-credentials.json` is on your machine where you want to run Lightly Worker. 
+    We need to access the private container registry of Lightly. You received 
+    a :code:`container-credentials.json` file from your account manager.
 
-For example, to authenticate  as non-root user you would run 
+2. Authenticate your docker account
+    To be able to download the Docker image you need to log in with these credentials.
```
Comment (japrescott) [inline docs/source/docker/getting_started/setup.rst:81]: download the Docker image -> download docker images  of Lightly

File: docs/source/docker/getting_started/setup.rst [hunk 5]
```diff
@@ -42,83 +42,82 @@ See :ref:`rst-installing` for details.
...
+1. Make sure :code:`container-credentials.json` is on your machine where you want to run Lightly Worker. 
+    We need to access the private container registry of Lightly. You received 
+    a :code:`container-credentials.json` file from your account manager.
 
-For example, to authenticate  as non-root user you would run 
+2. Authenticate your docker account
+    To be able to download the Docker image you need to log in with these credentials. 
 
-.. code-block:: console
+    The following command will authenticate your installed Docker account.
```
Comment (japrescott) [inline docs/source/docker/getting_started/setup.rst:83]: will authenticate your installed Docker account -> will authenticate yourself to gain access to the Lightly docker images

File: docs/source/docker/getting_started/setup.rst (hunk 3 above)
Comment (IgorSusmelj) [reply docs/source/docker/getting_started/setup.rst:68]: Added at the beginning of the block that we recommend Linux

**Pull request merged**